diff --git a/ya_market/api/requestor_api.py b/ya_market/api/requestor_api.py
index bb4db33..f832328 100644
--- a/ya_market/api/requestor_api.py
+++ b/ya_market/api/requestor_api.py
@@ -270,7 +270,7 @@ class RequestorApi(object):
             body=body_params,
             post_params=form_params,
             files=local_var_files,
-            response_type='list[OneOfProposalEventPropertyQueryEvent]',  # noqa: E501
+            response_type='list[ProposalEvent]',  # noqa: E501
             auth_settings=auth_settings,
             async_req=local_var_params.get('async_req'),
             _return_http_data_only=local_var_params.get('_return_http_data_only'),  # noqa: E501
@@ -774,7 +774,7 @@ class RequestorApi(object):
                                  number provided, it will be total request
                                  timeout. It can also be a pair (tuple) of
                                  (connection, read) timeouts.
-        :return: list[OneOfDemand]
+        :return: list[Demand]
                  If the method is called asynchronously,
                  returns the request thread.
         """
@@ -853,7 +853,7 @@ class RequestorApi(object):
             body=body_params,
             post_params=form_params,
             files=local_var_files,
-            response_type='list[OneOfDemand]',  # noqa: E501
+            response_type='list[Demand]',  # noqa: E501
             auth_settings=auth_settings,
             async_req=local_var_params.get('async_req'),
             _return_http_data_only=local_var_params.get('_return_http_data_only'),  # noqa: E501
